velox/common/testutil/CMakeLists.txt

```diff
@@ -15,6 +15,10 @@
 velox_add_library(velox_test_util ScopedTestTime.cpp TestValue.cpp)
 velox_link_libraries(velox_test_util PUBLIC velox_exception)
 
+velox_add_library(velox_test_output_matcher OutputMatcher.cpp)
+velox_link_libraries(velox_test_output_matcher PUBLIC Folly::folly GTest::gtest
+                                                      re2::re2)
```

Comment:
Like this it will still fail the build due to missing GTest when VELOX_BUILD_TESTING is not set. But because of the refactor you can just move it into the `if` and that should be fine (it's just used in tests right?)